optuna/dashboard.py

```diff
@@ -255,11 +257,11 @@ def update_callback(self):
             self.all_trials_widget.update(current_trials, new_trials)
 
 
+@experimental("0.1.0")
 def _show_experimental_warning():
     # type: () -> None
 
-    logger = optuna.logging.get_logger(__name__)
-    logger.warning("Optuna dashboard is still highly experimental. Please use with caution!")
+    warnings.warn("Optuna dashboard is still highly experimental. Please use with caution!")
```

Comment:
The message shown in the console is as follows:

```python
optuna/_experimental.py:63: ExperimentalWarning:

_show_experimental_warning is experimental (supported from v0.1.0). The interface can change in the future.

optuna/dashboard.py:264: UserWarning:

Optuna dashboard is still highly experimental. Please use with caution!
```

1) I think the target of experimental warning is  the dashboard, not `_show_experimental_warning`
2) The warning was divided into two. I think multiple warnings for a single issue may be confusing. Do we add the `text` argument to `@experimental` like `@deprecated`?

Reply:
Thanks for your comments. I totally agree with you.
1) I will specify the `name` argument for the experimental decorator.
2) I will remove the second warning because it is sufficient here to note the default message of the experimental decorator. BTW, it might be good to give the `text` argument to the experimental decorator.